README.md

```diff
@@ -240,8 +226,8 @@ You can inject this fake SMS into the emulated phone by following these steps.
 It is advisable that you pre-populate the emulated phone with the SMS _before_ running the sample app.
 
 > Once the app gets the permissions form the user it will instantly start sending existing SMS messages to the Pngme system. This results in messages being seen way sooner than SMS received after the app was installed.
-> 
-> As stated before, the daemon is processing new messages every 30 minutes, so the new feed messages will take at least 30 minutes to appear in the webconsole.
+>
+> The daemon is processing new messages every 30 minutes, so the new feed messages will take at least 30 minutes to appear in the webconsole.
```

Comment:
For me, it would read better if it was stated `a minimum of 30 minutes`. Or even `a minimum of 45(?) minutes from new feed to appearing in the dashboard.

Reply:
Minimum can be ambiguous with frequencies (can be interpreted as more frequent or less frequent than 30 min). I'm open to suggestions but since it isn't related to the content updates here, let's consider improving in subsequent changes?